raster/r.texture/benchmark/benchmark_rtexture_window.py

```diff
@@ -0,0 +1,82 @@
+"""Benchmarking of r.texture
+
+@author Aaron Saw Min Sern
+        Chung-Yuan Liang
+"""
+
+from grass.exceptions import CalledModuleError
+from grass.pygrass.modules import Module
+from subprocess import DEVNULL
+
+import grass.benchmark as bm
+
+
+def main():
+    results = []
+    mapsize = 1e6
+    window_sizes = [3, 9, 15, 27]
+
+    for window in window_sizes:
+        benchmark(int(mapsize**0.5), window, f"r.texture_{window}x{window}", results)
+
+    bm.nprocs_plot(
+        results,
+        title="r.texture -a speedup",
+        metric="speedup",
+    )
+    bm.nprocs_plot(
+        results,
+        title="r.texture -a efficiency",
+        metric="efficiency",
+    )
+
+
+def benchmark(size, window, label, results):
+    reference = "r_texture_reference_map"
+    basenames = [
+        "ASM",
+        "Contr",
+        "Corr",
+        "Var",
+        "IDM",
+        "SA",
+        "SV",
+        "SE",
+        "Entr",
+        "DV",
+        "DE",
+        "MOC-1",
+        "MOC-2",
+    ]
```

Comment:
This can be removed, in the other benchmark too.

Reply:
removed by https://github.com/OSGeo/grass/pull/3857/commits/8cd45a386ac2e318985299e6e844769aa75b64dd